packages/terra-popup/src/terra-dev-site/test/popup/BoundedNoHeaderPopup.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Popup from '../../../Popup';
+import BoundedNoHeaderContent from '../common/BoundedNoHeaderContent';
+
+class BoundedPopup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.setParentNode = this.setParentNode.bind(this);
+    this.getParentNode = this.getParentNode.bind(this);
+    this.state = { open: true };
+  }
+
+  componentDidMount() {
+    this.forceUpdate();
+  }
+
+  setButtonNode(node) {
+    this.buttonNode = node;
+  }
+
+  getButtonNode() {
+    return this.buttonNode;
+  }
+
+  setParentNode(node) {
+    this.parentNode = node;
+  }
+
+  getParentNode() {
+    return this.parentNode;
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  render() {
+    return (
+      <div id="test-popup-area" style={{ height: '200px', width: '200px', background: 'aliceblue' }} ref={this.setParentNode}>
+        <Popup
+          boundingRef={this.getParentNode}
+          classNameArrow="test-arrow"
+          classNameContent="test-content"
+          contentHeight="240"
+          contentWidth="320"
+          isOpen={this.state.open}
+          onRequestClose={this.handleRequestClose}
+          targetRef={this.getButtonNode}
+          isHeaderDisabled
+        >
+          <BoundedNoHeaderContent />
```

Comment:
It was listed as an error in console, but in actuality it acted like a warning. It ended up passing a prop to the children but changed the prop so that instead of getting
```
<div closeButtonRequired> I am some popup content </div>
```
it becomes
```
<div closebuttonrequired> I am some popup content </div>
```

We can remove the warning by just having the the prop always be the lowercase form "closebuttonrequired" or having some code figure out if the child is React or not.

@dkasper-was-taken your opinion?

Also the error/warning from console 
> Warning: React does not recognize the `closeButtonRequired` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `closebuttonrequired` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Reply:
The warning in intended. The values cloned on the child content are part of the contract when using it's isHeaderDisabled, contentHeight or contentWidth. Consumers need an indication that their component now needs to be responsive or present a close button. So the consumer should be warned that they are failing to correctly implement the popup.